defaults/main.yml

```diff
@@ -12,62 +12,21 @@ owncloud_domain: [ 'owncloud.{{ ansible_domain }}' ]
 
 # --- ownCloud source and deployment ---
 
+owncloud_edition: community
+owncloud_release: 8.0
+owncloud_apt_repo_base: 'http://download.opensuse.org/repositories/isv:/ownCloud:'
+
 # User and group that will be used for ownCloud instance
-owncloud_user: 'owncloud'
-owncloud_group: 'owncloud'
-owncloud_home: '/srv/users/{{ owncloud_user }}'
+owncloud_user: 'www-data'
+owncloud_group: 'www-data'
+owncloud_home: '/var/www/owncloud'
+
 
 # Path where ownCloud data/ directory and files are stored
-owncloud_data_path: '/srv/lib/{{ owncloud_user }}/{{ owncloud_domain[0] }}/data'
+owncloud_data_path: '{{ owncloud_home }}/data' #{{ owncloud_domain[0] }}'
```

Comment:
The idea here was to have the ability to host multiple separate ownCloud instances on the same host using different domains, but I suppose having separate containers can accomplish the same job. Also, this path should be specified using `'{{ ansible_local.root.data + "/owncloud" }}'`, which would by default be set to `/srv/owncloud/`.